bandyer-android-design/src/main/res/values/upload_image.xml

```diff
@@ -44,7 +44,7 @@
         <item name="cardUseCompatPadding">true</item>
     </style>
 
-    <style name="BandyerSDKDesign.UploadImage.Day" parent="BandyerSDKDesign.UploadImage">
+    <style name="BandyerSDKDesign.UploadImage.Day" parent="BandyerSDKDesign.BottomSheetDialog.Whiteboard.UploadImage">
```

Comment:
BandyerSDKDesign.UploadImage.Day should be refactored to BandyerSDKDesign.BottomSheetDialog.Whiteboard.UploadImage.Day